app/javascript/packs/followButtons.js

```diff
@@ -21,6 +21,20 @@ function addButtonFollowText(button, style) {
   }
 }
 
+/**
+ * Sets the aria-label of the button
+ *
+ * @param {HTMLElement} button The Follow button to update.
+ * @param {string} followType The followableType of the button.
+ * @param {string} followName The name of the followable to be followed.
+ */
+function addAriaLabelToButton(button, followType, followName) {
```

Comment:
This is great, but at the moment this is only being set on the initial page load, and we will need to update the aria-label when the user clicks on it too, e.g. 

```Follow ${followType.toLowerCase()}: ${followName}```

will need to become

```Unfollow ${followType.toLowerCase()}: ${followName}```

when the button is updated to the "Following state". Similarly the frontend has some code to change a User follow button text to "Follow back" when appropriate, and we would need to then also update the `aria-label` to `Follow ${followType.toLowerCase()} back: ${followName}`

Check out `addButtonFollowText` and `addButtonFollowingText` for where this happens currently